index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>microverse_trails_with_milenitic</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="body">
+        <div id="body-wrapper">
+    <div id="main">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </div>
+    <div id="form">
+        <form id="survey-form">
```

Comment:
@Santosh-Konappanavar, You have politely given the reasons for the first part while reviewing, which is great. :+1: 

- [ ] For the last par of the review, `And also I think, the header tag would be put always better in the top position in the body.` Kindly use reasoning the same way as the first part, and add any reference links, so that students can get any hints for the changes.